salt/haproxy/haproxy.manifest

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: haproxy
+  namespace: kube-system
+  labels:
+    name: haproxy
+spec:
+  containers:
+  - name: haproxy
+    resources:
+      requests:
+        memory: 128Mi
+      limits:
+        memory: 128Mi
+    image: haproxy:1.7.5
+    volumeMounts:
+    - mountPath: /usr/local/etc/haproxy
+      name: haproxy-cfg
+  volumes:
+  - hostPath:
+      path: /etc/haproxy
```

Comment:
Only Salt should be writing to that - it's intention is to contain the haproxy configuration only.